plugin/src/main/java/org/owasp/benchmarkutils/score/BenchmarkScore.java

```diff
@@ -98,42 +79,16 @@ public class BenchmarkScore extends AbstractMojo {
     // files. These affect overall scorecard generation. These were the original command line params
     // to scorecard generation.
 
-    // Indicates that the names of Commercial tools should be anonymized
-    static boolean anonymousMode = false;
-    // Indicates that the results of Commercial tools should be suppressed. Only show their
-    // averages.
-    public static boolean showAveOnlyMode = false;
-    // The name of the tool to 'focus' on, if any
-    private static String focus = "none";
-    // This is used to indicate that results from multiple versions of a test suite are included in
-    // these results. Each set in their own directory with their associated expectedresults file.
-    public static boolean mixedMode = false;
-
-    private static String expectedResultsFileName;
-    private static String resultsFileOrDirName;
-
     // Customizations of these can be set via scorecardconfig.yaml files
     // This affects details within whatever type of scorecards are being generated
 
```

Comment:
Doesn't this comment either need to be updated or moved as it seems to be left hanging there without context.

Reply:
Oh yeah, missed them. Will move them this weekend.

Reply:
Done